### ISSUE TITLE
New usage should be recommended in messages

### ISSUE THREAD

**Issue by sguptatw** [getgauge/gauge#689]
**Expected behavior**
When we give a message to the user for running a command, it should be the new usage.

**Actual behavior**
* Run specifications with "gauge specs/" message to be changed. This occurs in the initialization of a project

* The check updates message should be updated. This occurs during gauge run and when there are updates.

**Version**
```
Gauge version: 0.9.0.nightly-2017-06-28

Plugins
-------
csharp (0.10.1)
html-report (4.0.1.nightly-2017-06-27)
java (0.6.5.nightly-2017-06-28)
ruby (0.4.1)
spectacle (0.1.2)
xml-report (0.2.0)
```

**Comment by sguptatw** [comment]
`gauge update -c` still gives the old usage in the message
```
csharp    		0.10.1    	Run 'gauge --update csharp'
```

**Comment by sguptatw** [comment]
`gauge --docs abc` still prints the old usage
[DEPRECATED] This usage will be removed soon. Run gauge help --legacy for more info.
Error starting plugin abc. Failed to get plugin.json. Plugin abc is not installed. To install, run `gauge --install abc`.

**Comment by sriv** [comment]
Templates will not be fixed till we can ensure that users of old version of gauge are not affected. New commands are not available prior to gauge 0.9.0.
